docs/language-server/protocol-language-server.md

```diff
@@ -490,26 +492,55 @@ interface SuggestionEntryAtom {
   /** The module name where the atom is defined. */
   module: string;
 
-  /** The list of arguments. */
-  arguments: SuggestionEntryArgument[];
+  /** The list of type parameters. */
+  params: SuggestionEntryArgument[];
 
   /** The type of an atom. */
   returnType: string;
 
+  /** The fully qualified module name re-exporting this type. */
+  reexport?: string;
+
   /** The documentation string. */
   documentation?: string;
 
-  /** The fully qualified module name re-exporting this module. */
+  /** The rendered HTML of the documentation string. */
+  documentationHtml?: string;
+
+  /** The documentation string divided into sections. */
+  documentationSections?: DocSection[];
+}
+
+interface Constructor {
+  /** The external id. */
+  externalId?: UUID;
+
+  /** The atom name. */
+  name: string;
+
+  /** The module name where the atom is defined. */
+  module: string;
```

Comment:
1. Docs still use `atom` term.
2. Isn't module entirely deducible from `returnType` field? Is there a possibility of "Extension constructors"?

Reply:
Good catch, I updated all `atom` references in the doc.

The `module` can probably be deducible from `returnType` right now. But later when we start supporting more complex types, it might not be the case. I'd leave this field as is